### PR TITLE
internal/docker: add docker opener

### LINKED ISSUE #4: support docker:// opener
The docker:// opener isn't enabled in this codebase.

### PR DESCRIPTION
**Pull request by Fraser-Isbester**
**Summary:** Adds docker opener for entprint.

closes #4 

```shell
entprint --dir="./internal/example/ent/schema"  --dev="docker://postgres"
```

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 name: test
-on:
-  push:
```
Comment (Fraser-Isbester) [inline .github/workflows/test.yaml:3]: I can revert this, let me know.

File: internal/docker/docker.go [hunk 2]
```diff
@@ -0,0 +1,349 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package docker
+
+import (
```
Comment (rotemtam) [inline internal/docker/docker.go:9]: add a reference to the original file with credit please ?

**Comment by rotemtam** [comment]
Hey @Fraser-Isbester test for MySQL still red 🙏

**Comment by Fraser-Isbester** [comment]
> Hey @Fraser-Isbester test for MySQL still red 🙏

Went to sleep 😴, but I'll poke at it a bit more today. I'll draft the pr till I figure it out!

**Comment by Fraser-Isbester** [comment]
@rotemtam good to go. [CI is passing for me now](https://github.com/Fraser-Isbester/entprint/actions/runs/3609830732).

**Review by rotemtam — APPROVED** [review]

**Comment by rotemtam** [comment]
Thanks for working on this @Fraser-Isbester !

**Pull request merged**